tests/caasp/stack_update.pm

```diff
@@ -36,6 +35,14 @@ sub check_update_changes {
 
     # Kubernetes checks
     assert_script_run "kubectl cluster-info";
+    assert_script_run "kubectl cluster-info > cluster.after_update";
+    if (script_run "diff -Nur cluster.before_update cluster.after_update") {
+        record_soft_failure "old kubeconfig cannot see DEX - bsc#1081337";
+        switch_to 'velum';
+        download_kubeconfig;
```

Comment:
No need to switch to xterm and update ~/.kube/config, it's part of download_kubeconfig now

Reply:
I've removed it ;)